editor-components/Image_gallery/image_gallery_7/image-gallery7.tsx

```diff
@@ -0,0 +1,490 @@
+                            },
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                             
+                        ]
+                    },
+                    {
+                        type: "array",
+                        key: "card",
+                        displayer: "Card 12",
+                        value: [
+                            { 
+                                type: "image",
+                                key: "image",
+                                displayer: "Image 12",
+                                value:"https://storage.googleapis.com/download/storage/v1/b/hq-composer-0b0f0/o/669e01732f8a5b002ce6018d?alt=media"
+
+                                , 
+                            },
+                            {
+                                type: "string",
+                                key: "title",
+                                displayer: "title",
+                                value: "National Aquarium Dubai",
+                            },
+                            {
+                                type: "string",
+                                key: "subtitle",
+                                displayer: "subtitle",
+                                value: "Brutalism, Portraits"
+                            }
+                            ,
+                            {
+                                type:"color",
+                                key:"backgroundColor",
+                                displayer:"Background Color",
+                                value:"#f1f1e8"
+                            }
+                        ]
+                    }
+            ]
+        });
+
+        this.addProp({
+            type:"number",
+            key:"cardRepeatTime",
+            displayer:"Repeat Count",
+            value:2
+        });
+
+
+     
+    }
+    
+
+      getName(): string {
+        return "Image Gallery 7";
+      }
+    
+      render() {
+  const gallery = this.getPropValue("gallery") as CardType[];
+  const repeatCount = this.getPropValue("cardRepeatTime") as number;
+  const cardBackgroundColor = this.getPropValue("backgroundColor") as string || "#f1f1e8";
+ 
+  const repeatedGallery = Array.from({ length: repeatCount }).flatMap(() => gallery);
+
+  return (
+    <div
+      className={this.decorateCSS("imageGallery7")}
+      style={{
+        "--composer-card-background": cardBackgroundColor,
+      } as React.CSSProperties}
+    >
+      <div className={this.decorateCSS("content")}>
+        {Array.from({ length: 4 }).map((_, columnIndex: number) => (
+          <div key={columnIndex} className={`${this.decorateCSS("column")} ${this.decorateCSS(`column-${columnIndex}`)}`}>
```

Comment:
find a better way for this -> ```${this.decorateCSS(`column-${columnIndex}`)}```
```suggestion
          <div key={columnIndex} className={`${this.decorateCSS("column")} ${this.decorateCSS(`column-${columnIndex}`)}`}>
```